Helper/Config.php

```diff
@@ -623,4 +628,36 @@ public function getGoogleTrackOnCheckoutStart()
             ScopeInterface::SCOPE_STORE
         );
     }
+
+    /**
+     * Get Additional Config string
+     *
+     * @return  string
+     */
+    private function getAdditionalConfigString() {
+        return $this->getScopeConfig()->getValue(
+            self::XML_PATH_ADDITIONAL_CONFIG,
+            ScopeInterface::SCOPE_STORE
+        ) ?: '{}';
+    }
+
+    /**
+     * Get Additional Config object
+     *
+     * @return  \stdClass
+     */
+    private function getAdditionalConfigObject() {
+        return json_decode($this->getAdditionalConfigString());
+    }
+
+    /**
+     * Get Additional Config property
+     *
+     * @param $name
+     * @return mixed
+     */
+    private function getAdditionalConfigProperty($name) {
+        $config = $this->getAdditionalConfigObject();
+        return @$config->$name;
```

Comment:
Maybe better way to return array instead of the StdClass from $this->getAdditionalConfigObject(), is it?

I suggest make like `json_decode($this->getAdditionalConfigString(), true)` after that we have assoc array in `getAdditionalConfigObject` method.

Then here we can check is set this key and not use the `@`.

Reply:
@YevhenShBolt I prefer using objects later when we actually use this method and the configuration options stored like this. They could be nested objects themselves. It is less verbose and somehow more natural. On the other hand, it may be just a personal preference.

🡣🡣🡣 fixed the brackets